File: WarnIt-Address/src/main/java/br/com/warnit/address/WarnItAddressApplication.java
package br.com.warnit.address;

import br.com.warnit.address.model.domain.Logradouro;
import br.com.warnit.address.repository.LogradouroRepository;
import br.com.warnit.address.util.builder.LogradouroBuilder;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.CommandLineRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.loadbalancer.LoadBalanced;
import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.client.RestTemplate;

@SpringBootApplication
@EnableEurekaClient
public class WarnItAddressApplication implements CommandLineRunner {

	@Autowired
	private LogradouroRepository logradouroRepository;

	public static void main(String[] args) {
		SpringApplication.run(WarnItAddressApplication.class, args);
	}

	@Override
	public void run(String... args) throws Exception {
		/*Logradouro Completo*/
		Logradouro logradouro = LogradouroBuilder.create().get();
		logradouro = logradouroRepository.save(logradouro);
	}
}
@Configuration
class RestTemplateConfig {

	// Create a bean for restTemplate to call services
	@Bean
	@LoadBalanced        // Load balance between service instances running at different ports.
	public RestTemplate restTemplate() {
		return new RestTemplate();
	}
}

File: WarnIt-Address/src/main/kotlin/br/com/warnit/address/WarnItAddressApplication.kt
package br.com.warnit.address

import br.com.warnit.address.service.LocalidadeService
import org.springframework.boot.CommandLineRunner
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication
@EnableDiscoveryClient
class WarnItAddressApplication(private val localidadeService: LocalidadeService) : CommandLineRunner {

	override fun run(vararg args: String?) {

	}
}

fun main(args: Array<String>) {
	runApplication<WarnItAddressApplication>(*args)
}
